Tilesheets.body.php

```diff
@@ -34,7 +37,7 @@ public function __construct($options, Parser &$parser) {
 
 		TilesheetsError::log(wfMessage('tilesheets-log-prepare')->params($size, $item, $mod)->text());
 
-		$dbr = wfGetDB(DB_REPLICA);
+		$dbr = MediaWikiServices::getInstance()->getDBLoadBalancer()->getConnection( DB_REPLICA );
```

Comment:
I know it's not great, all other methods here are static thats why I just left it as is since it fills some static variables with data  during autoload of classes. I think we might consider greater refactor during next upgrade. With especially moving some code to appropriate Model classes instead of having spaghetti code here and there with static calls.